onnx2kerastl/constant_layers.py

```diff
@@ -10,3 +13,10 @@ def convert_constant(node, params, layers, lambda_func, node_name, keras_name):
     :return: None
     """
     layers[node_name] = params['value']
+
+
+def convert_constant_of_shape(node, params, layers, lambda_func, node_name, keras_name):
+    value = params.get('value')
+    if value is None:
+        raise NotImplementedError("ConstantOfShape should have a value param")
```

Comment:
I see in the docs that the value param is optional and the default is 0

Reply:
This is not the purpose - the default value is OK IMO, only checking validity of the data so I won't multiply by None if the onnx config is somehow corrupted